ModelBase/src/nodes/Node.h

```diff
@@ -311,6 +311,11 @@ class MODELBASE_API Node
 		NodeType* firstAncestorOfType();
 
 		/**
+		 * Returns the first Ancestor for which typeName() is equal to \a typeName if there is one, otherwise null.
```

Comment:
Changed to 
Returns the first Ancestor for which typeName() matches with \a typeMatch, if there is one, otherwise null.